prbt_hardware_support/include/prbt_hardware_support/modbus_adapter_operation_mode.h

```diff
@@ -62,7 +62,7 @@ class ModbusAdapterOperationMode : public AdapterOperationMode
   std::unique_ptr<FilterPipeline> filter_pipeline_;
 
 private:
-  static constexpr unsigned int MODBUS_API_VERSION_REQUIRED{ 2 };
+  static constexpr unsigned int MODBUS_API_VERSION_REQUIRED{ 3 };
```

Comment:
Maybe move modbus version into a common header to avoid duplication across multiple adapters

Reply:
Done